### PR TITLE
Add CLI troubleshooting instructions

### PR DESCRIPTION
**Pull request by Alton09**
## Description

Added CLI troubleshooting instructions.

## Breakdown

- Updated README to include CLI troubleshooting instructions

## Validation

N/A

## Additional Notes

N/A

## Submission Checklist
 - [x] The source has been evaluated for semantic versioning changes and are reflected in ```versionName``` under `app/build.gradle`
 - [x] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,19 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
...
+
+If any errors occur after running the `twilio rtc:apps:video:deploy --authentication passcode` command, then run the following commands:
+
+```
+twilio plugins:update
+twilio rtc:apps:video:delete
+twilio rtc:apps:video:deploy --authentication passcode
+```
+
+The commands in the above snippet updates the rtc plugin to the latest version, deletes any problematic authentication servers, and deploys a new authentication server.
```
Comment (timrozum) [inline README.md:53]: There seem to be 2 different styles used. Here we list out all the commands and then describe what they all do. In the deploy section we specify the command and describe what it does on the same line as needed. Not a big deal but probably better to be consistent unless there is a reason not to. It seems like the "Run abc command to do blah" approach works in more situations. Again pretty minor but thinking probably best to follow other pattern.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,19 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
 The passcode will expire after one week. To generate a new passcode:
 
 1. Run `twilio rtc:apps:video:delete`.
-1. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+2. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+
+#### Troubleshooting The Twilio CLI
+
+If any errors occur after running the `twilio rtc:apps:video:deploy --authentication passcode` command, then run the following commands:
```
Comment (timrozum) [inline README.md:45]: Should we be less specific and just say hey if you run into (basically any) CLI problem you can try these commands? Mainly thinking we could avoid the duplicate maintenance of this command syntax. If syntax changes not sure we will remember to change here. Not a big deal though.

Comment (aaalaniz) [inline README.md]:
```suggestion
If any errors occur after running a [Twilio CLI RTC Plugin](https://github.com/twilio-labs/plugin-rtc) command, then try the following steps.
```

File: README.md [hunk 3]
```diff
@@ -38,7 +38,19 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
...
+
+```
+twilio plugins:update
+twilio rtc:apps:video:delete
+twilio rtc:apps:video:deploy --authentication passcode
+```
```
Comment (aaalaniz) [inline README.md]:
```
# Update to the latest RTC Plugin
twilio plugins:update

# Delete any existing applications
twilio rtc:apps:video:delete

# Retry deploy
twilio rtc:apps:video:deploy --authentication passcode
```

File: README.md (hunk 1 above)
Comment (aaalaniz) [inline README.md]:
```suggestion
```

**Comment by Alton09** [comment]
Good feedback @timrozum ! I'll make those changes.

File: README.md [hunk 4]
```diff
@@ -38,7 +38,15 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
...
 1. Run `twilio rtc:apps:video:delete`.
-1. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+2. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+
+#### Troubleshooting The Twilio CLI
+
+If any errors occur after running the [Twilio CLI commands](#deploy-twilio-access-token-server), then run the following commands:
+
+1. Run `twilio plugins:update` to update the rtc plugin to the latest version.
+1. Run `twilio rtc:apps:video:delete` to delete any problematic authentication servers.
```
Comment (aaalaniz) [inline README.md:48]:
```suggestion
1. Run `twilio rtc:apps:video:delete` to delete any existing authentication servers.
```

File: README.md [hunk 5]
```diff
@@ -38,7 +38,15 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
 The passcode will expire after one week. To generate a new passcode:
 
 1. Run `twilio rtc:apps:video:delete`.
-1. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+2. Run `twilio rtc:apps:video:deploy --authentication passcode`.
```
Comment (timmydoza) [inline README.md:41]: I don't think we need two steps here. 

"The passcode will expire after one week. To generate a new passcode, run `twilio rtc:apps:video:deploy --authentication passcode --override`."

**Pull request merged**